repository.json

```
@@ -1,4 +1,5 @@
 {
   "name": "Photons",
-  "url": "https://github.com/delfick/photons-homeassistant"
+  "url": "https://github.com/delfick/photons-homeassistant",
+  "maintainer": "delfick"
```

Actually, better would be

```suggestion
  "maintainer": "Stephen Moore <github@delfick.com>"
```

---

Yup, thats better